templates/partials/sender_info_col.mjml

```diff
@@ -0,0 +1,30 @@
+<mj-column css-class="senderInfoCol" width=90% >
+    <mj-raw>
+        <tr>
+            <td>
+                <div class="postNameAndTime">
```

Comment:
Can't we put this class on the `td` tag?

Reply:
We can put it wherever we want.

Reply:
I guess we can remove the extra div, but this isn't blocking anything so up to you. Thanks.